src/h5web/visualizations/shared/utils.ts

```diff
@@ -31,3 +32,17 @@ export function computeVisSize(
     ? { width: availableHeight * aspectRatio, height: availableHeight }
     : { width: availableWidth, height: availableWidth * aspectRatio };
 }
+
+export function extendDomain(domain: Domain, extendFactor: number): Domain {
+  const [min, max] = domain;
+  const extension = (max - min) * extendFactor;
+
+  return [min - extension, max + extension];
+}
+
+export function findDomain(data: number[]): Domain | undefined {
+  const extentD3 = extent(data);
```

Comment:
`domain` ?

Reply:
I fear `domain` is too generic and may shadow some variables as we are moving a lot of domains around...

Reply:
it's just a local variable, though

Reply:
Yes, my fear was unfounded: it is all right everywhere else in fact.

Reply:
Oh sorry for the confusion, I meant the local variable `extentD3`, not the function name! I liked `findDomain` 😅